app/controllers/errors_controller.rb

```diff
@@ -44,7 +44,7 @@ def update
   def notify_subscribers
     unless params[:message].blank?
       @message = Message.create( content: params[:message], issue_id: @error.issues.last.id )
-      @error.website.users.each do |user|
+      @error.website.subscribers.each do |user|
         GroupedIssueMailer.notify_subscriber(@error, user, current_user, @message).deliver_later
```

Comment:
I think we need to change the each argument too from `user -> subscriber` just so we can be correct and not give false impressions.